tests/integration/utils/runCommand.ts

```diff
@@ -2,7 +2,9 @@ import * as childProcess from "child_process";
 
 export const runCommand = (command: string): Promise<CommandResult> => {
   return new Promise((resolve, reject: (err: CommandError) => void) => {
-    childProcess.exec(`node ./build/src/index.js ${command}`, (error, stdout, stderr) => {
+    childProcess.exec(`node ./build/src/index.js ${command}`, {
+      env: { ...process.env, MOCKED_DATE: process.env.MOCKED_DATE },
```

Comment:
what's that brother 
<img width="299" alt="image" src="https://github.com/kontent-ai/data-ops/assets/54802833/520667c1-d28e-4541-b293-493340f5df06">

Reply:
brother eww, leftover from the past. fixed